keychain/btcwallet.go

```diff
@@ -473,7 +473,7 @@ func (b *BtcWalletKeyRing) SignMessageSchnorr(keyLoc KeyLocator,
 	}
 
 	if len(taprootTweak) > 0 {
-		privKey = txscript.TweakTaprootPrivKey(privKey, taprootTweak)
+		privKey = txscript.TweakTaprootPrivKey(*privKey, taprootTweak)
```

Comment:
hmmm so the `privKey` can be mutated by others?

Reply:
Yeah, tweaking ended up mutating the key. Schnorr signing too, so we'll want https://github.com/btcsuite/btcd/pull/1919 to also be included (which is why the itest would also currently fail, independent of #7126).